### DISCUSSION TITLE
Please help! I want to convert the unit "W" to "A" at the position as in the picture, I hope to get help. Thank you very much.

### DISCUSSION THREAD

**Discussion by thaibacgiang** [slipx06/sunsynk-power-flow-card#594] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Please help! I want to convert the unit "W" to "A" at the position as in the picture, I hope to get help. Thank you very much.
![743d349f6123dc7d8532](https://github.com/user-attachments/assets/f870e98a-445e-4f67-8943-8f091eb8a06c)

### Expected behaviour

Please help! I want to convert the unit "W" to "A" at the position as in the picture, I hope to get help. Thank you very much. 
![743d349f6123dc7d8532](https://github.com/user-attachments/assets/3249c8b8-9ba1-49a4-ba2a-0d23d521abb1)

### Possible Solutions

Please help! I want to convert the unit "W" to "A" at the position as in the picture, I hope to get help. Thank you very much.

### Mode

lite

### Context / Reason

Please help! I want to convert the unit "W" to "A" at the position as in the picture, I hope to get help. Thank you very much.

**Comment by slipx06** [comment]
Hi, That placeholder requires a sensor that provides power (W). The easiest fix is for you to create a fork and update as per your pull request

**Comment by thaibacgiang** [comment]
> Hi, That placeholder requires a sensor that provides power (W). The easiest fix is for you to create a fork and update as per your pull request

How can I create an additional branch myself? Can the author please guide me? Thanks!

**Comment by slipx06** [comment]
Have a look here on how to setup a dev environment 

https://slipx06.github.io/sunsynk-power-flow-card/contribute/devcontainer.html

You can then build your own version with the required change.
